### PR TITLE
ref(ui): incidents design pass

### PR DESCRIPTION
**Pull request by ckj**
There's more i'd like to do here, but here's a quick initial pass:

Screen Shot 2019-06-18 at 2 12 12 PM

and

Screen Shot 2019-06-18 at 2 10 46 PM

#### Overall

- [x] Treat Incident as a proper noun in the app

#### Incidents Index

- [x] h1: kill vertical margin so it will center properly
- [x]  right align numerical cells

#### Incidents Detail

- Header
    - [x] Reorder header items to match order on index
    - [x] Make breadcrumb compete less with title
    - [x] 30px horizontal padding
    - [x] Make buttons smaller
        - [x] Bold Status button text
    - [x] Add a “Duration” metric?

### CODE REVIEW

File: src/sentry/static/sentry/app/components/dropdownControl.jsx [hunk 1]
```diff
@@ -104,7 +104,6 @@ const StyledDropdownButton = styled(
 )`
   z-index: ${p => p.theme.zIndex.dropdownAutocomplete.actor};
   white-space: nowrap;
-  font-weight: normal;
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/dropdownControl.jsx]: Is this what we want? https://percy.io/getsentry/sentry/builds/2055032/view/133452734/1280?browser=firefox&mode=diff

File: src/sentry/static/sentry/app/views/organizationIncidents/details/header.jsx [hunk 2]
```diff
@@ -44,7 +46,7 @@ export default class DetailsHeader extends React.Component {
           label={incident && <Status incident={incident} />}
           menuWidth="160px"
           alignRight
-          buttonProps={!incident ? {disabled: true} : null}
+          buttonProps={!incident ? {disabled: true, size: 'small'} : {size: 'small'}}
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationIncidents/details/header.jsx]: I would do something like this

```suggestion
          buttonProps={{size: 'small', disabled: !incident}}
```

File: src/sentry/static/sentry/app/views/organizationIncidents/list/index.jsx [hunk 3]
```diff
@@ -61,13 +61,12 @@ class OrganizationIncidentsList extends AsyncComponent {
...
-          <div>
+          <AlignRight>
             <Count value={incident.uniqueUsers} />
-          </div>
-          <div>
+          </AlignRight>
+          <AlignRight>
             <Count value={incident.totalEvents} />
-          </div>
+          </AlignRight>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationIncidents/list/index.jsx]: Can we give this less of a presentational name and more semantic, e.g. `NumericColumn` or something

File: src/sentry/static/sentry/app/views/organizationIncidents/list/index.jsx [hunk 4]
```diff
@@ -95,9 +94,9 @@ class OrganizationIncidentsList extends AsyncComponent {
             <TableLayout>
               <div>{t('Incident')}</div>
               <div>{t('Status')}</div>
-              <div>{t('Started')}</div>
-              <div>{t('Users affected')}</div>
-              <div>{t('Total events')}</div>
+              <div>{t('Duration')}</div>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationIncidents/list/index.jsx]: We have the absolute date there, will "duration" be confusing?

File: src/sentry/static/sentry/app/components/dropdownControl.jsx (hunk 1 above)
Comment (ckj) [reply src/sentry/static/sentry/app/components/dropdownControl.jsx]: Yep! The previous behavior was unintended and this change makes button font-weight consistent.

File: src/sentry/static/sentry/app/views/organizationIncidents/list/index.jsx (hunk 4 above)
Comment (ckj) [reply src/sentry/static/sentry/app/views/organizationIncidents/list/index.jsx]: Ah, looks like the column is now "Started," so i'll update this to match.

**Review by billyvg — APPROVED** [review]

**Comment by billyvg** [comment]
Incident details header is a bit crowded, otherwise lgtm

**Pull request merged**